Upgrade to v0.2.2 & signald properly & linter (#36)
YunoHost-Apps/mautrix_signal_ynh: doc/DISCLAIMER.md

## List of known public services

* Ask on one of the following rooms: #mautrix_yunohost:matrix.fdn.fr or #signal:maunium.net

## Bridging usage
** Note that several Signal and Matrix users can be bridged, each Signal account has its own bot administration room. If they are in a same Signal group, only one matrix room will be created. **

### Bridge a Signal user and a Matrix user
* First your Matrix user or Synapse Server has to be authorized in the Configuration of the bridge (see below)
* Then, invite the bot (default @signalbot:yoursynapse.domain) in this new Mautrix-Signal bot administration room.
  * If the Bot does bot accept, see the [troubleshooting page](https://docs.mau.fi/bridges/general/troubleshooting.html)

> * Send ``!sg help`` to the bot in the created room to know how to control the bot.

Reviewer: hum, okay, I actually thought `!sg didn't work`. However, I wanted to discourage people posting their phone numbers in public rooms.

See also [upstream wiki Authentication page](https://docs.mau.fi/bridges/python/signal/authentication.html)

#### Linking the Bridge as a secondary device

> * Type ``!sg link``

Reviewer: Samesies:
```suggestion
*  In the direct chat with the bot, type ``link``
```

* Open Signal App of your primary device
* Open Settings => Linked Devices => + => Capture the QR code with the camera
* By defaults, only conversations with very recent messages will be bridged
* Accept invitations to the bridged chat rooms

#### Registering the Bridge as a primary device

> * Type ``!sg register <phone>``, where ``<phone>`` is your phone number in the international format with no space, e.g. ``!sg register +33612345678``
> * Answer in the bot room with the verification code that you reveived in SMS.

Reviewer: Same here:
```suggestion
* Open the direct chat you have with the bot or create a new one
* Type ``register <phone>``, where ``<phone>`` is your phone number in the international format with no space, e.g. ``register +33612345678``
```

* Set a profile name with ``!sg set-profile-name <name>``

### Double puppeting
* Log in with ``login-matrix <access token>``
* After logging in, the default Matrix puppet of your Signal account should leave rooms and your account should join all rooms the puppet was in automatically.

### Relaybot: Bridge a group for several Matrix and several Signal users to chat together

> Not yet available

Reviewer: This is false.

```suggestion
* Create a room on the signal side
* Your bridged users will be invited on the Matrix side once they are invited on the Signal side
* You can invite more people over on the Matrix side
* Have one of the bridged users (who has the right permission) type `!sg set-relay` on the Matrix side. Their signal account will relay messages from other Matrix users
It is not yet possible to bridge to an existing signal room, or create a new signal room from the Matrix side.
```

Reviewer: I used relaybot functionality just fine using the instructions I wrote here.

I have never tried plumbing existing rooms, but I agree that would be interesting!
